webapp/src/actions.js

```diff
@@ -129,9 +130,17 @@ export const bump = (id) => async (dispatch, getState) => {
 };
 
 export function autocompleteUsers(username) {
-    return async (doDispatch) => {
-        const {data} = await doDispatch(UserActions.autocompleteUsers(username));
-        return data.users;
+    return async (doDispatch, getState) => {
+        const users = [];
+        const team = TeamSelector.getCurrentTeam(getState());
+        const {data} = await doDispatch(UserActions.autocompleteUsers(username, team.id));
+        data.users.forEach((user) => {
+            //Returning only non deleted users
+            if (user.delete_at === 0) {
+                users.push(user);
+            }
+        });
+        return users;
```

Comment:
This can be done with `filter`:

```suggestion
        return data.users.filter((user) => user.delete_at === 0);
```